### DISCUSSION TITLE
Navisworks Bridge?

### DISCUSSION THREAD

**Discussion by youngbrett** [iTwin/itwinjs-core#700] [Ideas] [answered]
Any thought of creating a bridge from Navisworks to iModelHubs?  General contractors in the building space often get Navis models and don't have the skill or time to use Revit, if they have a file from a subcontractor.

**Comment by timlawrence-bentley** [comment, marked as answer]
Hi Brett,

We have a connector (previously known as bridges) for Navisworks that is available as an EAP for some of our users. We hope to have it released for general use sometime in Q1 of this year.

Tim

**Reply by youngbrett** [reply]
Thx, @timlawrence-bentley.  Let me know if it's available for testing.

**Reply by timlawrence-bentley** [reply]
hi Brett,

What organization are you associated with? Can you send me your details via my Bentley email tim.lawrence@bentley.com?

Thanks,

Tim
